rollup/tracing/proof.go

```diff
@@ -103,46 +101,47 @@ func (t *ProofTracer) GetDeletionProofs() ([][]byte, error) {
 }
 
 // MarkDeletion mark a key has been involved into deletion
-func (t *ProofTracer) MarkDeletion(key []byte) {
+func (t *ProofTracer) MarkDeletion(key []byte) error {
```

Comment:
does adding an error return for interface compatibility? because I see in the function, it only returns nil when success and panic in error paths.

Reply:
yeah, looks unneeded. none of the callsites check the returned error either.